languageserver/server/server.go

```diff
@@ -753,12 +753,18 @@ func (s *Server) DocumentHighlight(
 
 	position := conversion.ProtocolToSemaPosition(params.Position)
 	occurrences := checker.Occurrences.FindAll(position)
+	if occurrences == nil {
+		return nil, nil
+	}
```

Comment:
If the result is nil, then the below condition should get executed. i.e:`len()` of a nil array is also zero.

It is safe to skip the nil check here and below, as `len()` and `range` expressions are nil-safe.